tests/test_gates.py

```diff
@@ -151,6 +152,8 @@ def test_gate_unitary_correct(op, op_name):
         pytest.skip("Skipping operation because it is a state preparation")
     if op == None:
         pytest.skip("Skipping operation.")
+    if op_name == "PSWAP" and device_name != "lightning.qubit":
+        pytest.skip("Skipping PSWAP test for non-lightning.qubit device.")
```

Comment:
```suggestion
    if op_name == "PSWAP" and device_name != "lightning.qubit":
        pytest.skip("Skipping PSWAP test for non-supported device.")
```

Reply:
Actually we shouldn't be skipping these tests anymore. Thanks for pointing this out! I removed these lines in a23463277d510ab0dcd76dc3e2d166a618a91d4a

Reply:
This is great! Thanks!